packages/jsii-pacmak/lib/targets/go/package.ts

```diff
@@ -107,6 +107,25 @@ export abstract class Package {
     this.emitTypes(context);
     code.closeFile(this.file);
 
+    if (this.types.length > 0) {
+      const initFile = this.file.replace(/(\.go)$/, '.init$1');
+      code.openFile(initFile);
+      code.line(`package ${this.packageName}`);
+      code.line();
+      code.open('import (');
+      code.line('"reflect"');
+      code.line();
+      code.line(`${JSII_RT_ALIAS} "${JSII_RT_MODULE_NAME}"`);
+      code.close(')');
```

Comment:
This seems like a reporting pattern in the Go code gen, maybe extract to a helper method on `Package`?

```ts
this.emitImports('reflect', `${JSII_RT_ALIAS} "${JSII_RT_MODULE_NAME}"`);
```

Or:

```ts
this.emitGoClause('imports', [ ... ]);
```